test/runtests.jl

```diff
@@ -3,6 +3,10 @@ using Test, Documenter
 
 @testset "tropical" begin
     include("tropical.jl")
+    include("tropical_andor.jl")
+    include("tropical_maxmul.jl")
+    include("tropical_minplus.jl")
+    include("tropical_maxplus.jl")
```

Comment:
This test seems to be redundant with `tropical.jl`. We just need a single line test in `tropical.jl` to show Tropical and TropicalMaxPlus are equivalent.

Reply:
Also, please separate these tests into different test sets.